curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153897c27f6334716ee5abe.md

```diff
@@ -7,7 +7,12 @@ dashedName: step-10
 
 # --description--
 
-Align your `.header` text in the center, give it a padding of `32px` on all sides, and set the background to `#E0DDDD`.
+Align your `.header` text in the center. Make the text uppercase using the `text-transform` property with `uppercase` as the value.
+
+Give it a padding of `32px` on all sides. Set the background to `#0a0a23` and the text to `#fff` as the color values.
```

Comment:
Not sure we really need to be this explicit for all the instructions. I think that is kind of the point of having the hints show the actual properties and values but letting the camper (when they should be able to) figure out the properties needed.

I'm also not crazy about the long-running sentence.

Reply:
I have updated it, my point was "looking at the forum, there are many people who ask for help but the answer in the hints".

I think there is something blocking hints from some users, maybe mobile devices, but that is another issue and it shouldn't affect this

Reply:
Why is it only the padding now that is in a code block? If they know how to set the background color and text color, surely they also know how to add padding? Otherwise, we might as well go back to what you had before with all the properties specified.

Also, I think the sentence is grammatically correct with or without the comma and I'd like to not have the punctuation near the code block if it can be avoided.

---

> I have updated it, my worry was "looking at the forum, there are many people who ask for help but the answer in the hints".

I agree that when we do not spoon-feed the camper it does result in more support threads. But I also worry that doing that too much too often might be a disservice to the camper's learning experience.

In any case, in my opinion, it should be as you first suggested with all the properties specified, or kept as it is now.

Reply:
> Why is it only the padding now that is in a code block?

A hint that the other two are properties

Reply:
Let me tag someone from the forum 👍

Reply:
Honestly, I don't think the change is needed.

If we do need to be more explicit I would go with your first suggestion with all the properties specified.

Reply:
I liked the hint, with code block the `padding` 😞. 

```
Give it a `padding` of `32px` on all sides. Set the background to `#0a0a23` and the text to `#fff` as the color values.
```

I'm kinda leaning with you, we don't need to hold their hand forever, otherwise people will start posting their actual project and not our steps, because they are too used to us holding their hand than they making projects.